client/dashboard/docs/ui-components.md

```diff
@@ -23,20 +23,11 @@ Relying on CSS overrides and hacks should be avoided as much as possible.
 
 ## Additional Layout Components
 
-In addition to the core components, the dashboard includes several reusable layout components that help structure the UI:
+In addition to the core components, the dashboard includes several reusable layout components that help structure the UI. They are found in the `/client/dashboard/components` directory:
 
-- **Page Layout** (`/client/dashboard/page-layout/`): The main container for every dashboard page.
-- **Menu** (`/client/dashboard/menu/`) and **Responsive Menu** (`/client/dashboard/responsive-menu/`): Reusable components to render mobile-friendly navigation menus.
-- **DataViews Card** (`/client/dashboard/dataviews-card/`): A card component for displaying data views.
-- **Overview Card** (`/client/dashboard/overview-card/`): Standard card for data display
-- **Header Bar** (`/client/dashboard/header-bar/`): A reusable header bar component for displaying a header bar (main and secondary headers).
+- **Page Layout**: The main container for every dashboard page.
+- **Menu** and **Responsive Menu**: Reusable components to render mobile-friendly navigation menus.
+- **DataViews Card**: A card component for displaying data views.
+- **Overview Card**: Standard card for data display
+- **Header Bar**: A reusable header bar component for displaying a header bar (main and secondary headers).
 
-## Accessibility Considerations
```

Comment:
I removed this part because there's a comprehensive a11y guidelines as part of Calypso Dev Docs and there's nothing specific to the new hosting dashboard in terms of a11y.